include/multipass/file_ops.h

```diff
@@ -39,6 +39,7 @@ class FileOps : public Singleton<FileOps>
     FileOps(const Singleton<FileOps>::PrivatePass&) noexcept;
 
     // QDir operations
+    virtual QDir current() const;
```

Comment:
I'm not sure if I'm missing it, but I didn't see anywhere in the tests where you inject this.  If you didn't, then this isn't necessary.

Reply:
`current()` is used [here](https://github.com/canonical/multipass/blob/f27a8bfc6c541f4fad92558ac450debb59d67f91/src/client/cli/cmd/exec.cpp#L75).

Reply:
Oh, right, in the tests I don't use it. I'll remove it then.